### PR TITLE
wavetool.py: add chirp wave generation

### PR DESCRIPTION
**Pull request by aiChaoSONG**
Usage:
wavetool.py -g chirp -A amplitude -F start_freq end_freq -D duration -C channel_count -P initial_phase -o file.wav

Chirp wave generated by: `wavetool.py -g chirp -F 400 4000 -D 4. -A 0.5 -o chirp.wav`
[chirp.zip](https://github.com/thesofproject/sof-test/files/5259598/chirp.zip)

### CODE REVIEW

**Comment by xiulipan** [comment]
Any update? @singalsu Could you have a look here for the algorithm

File: tools/wavetool.py [hunk 1]
```diff
@@ -81,6 +81,22 @@ def generate_wov():
...
+    ``chirp_rate = (f_end - f_start) / duration``
+
+    Returns
+    ----------
+    Real n-D Array with shape ``(duration * sample_rate, channel)``
+    """
+    time = np.arange(0, cmd.duration[0], 1.0 / cmd.sample_rate)
+    freq = cmd.freq[0] + np.arange(len(time)) * (cmd.freq[1] - cmd.freq[0]) / len(time)
+    data = cmd.amp[0] * np.sin(2 * np.pi * freq * time + cmd.phase[0])
+    return np.reshape(np.repeat(data, cmd.channel),[len(data), cmd.channel])
```
Comment (plbossart) [inline tools/wavetool.py:98]: generic comment on this file: you should look into generating different waveforms for each channel. Having the same chirp on all channels isn't always useful. Likewise having two frequencies for left and right sinewaves, or different amplitudes is VERY useful.

**Comment by xiulipan** [comment]
@aiChaoSONG Ping for update or close.

**Comment by aiChaoSONG** [comment]
chirp wave is currently not used, if needed, will reopen PR. close

**Pull request closed without merging**